.github/workflows/publish.yml

```diff
@@ -148,6 +148,22 @@ jobs:
           overwrite: true
           tag: ${{ inputs.tag || 'nightly' }} # This will fail if `inputs.tag` is not a tag (e.g. testing a branch)
 
+      - name: Get formatted date
+        id: date
+        if: ${{ inputs.tag == '' && inputs.publish || github.event_name == 'schedule' }}
+        run: echo "date=$(date '+%Y-%m-%d')" >> $GITHUB_OUTPUT
+
+      - name: Upload binaries to nightly release with date tag
+        uses: svenstaro/upload-release-action@v2
+        if: ${{ inputs.tag == '' && inputs.publish || github.event_name == 'schedule' }}
+        with:
+          repo_name: noir-lang/noir
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ./nargo-${{ matrix.target }}.zip
```

Comment:
Could we retain tar.gz extension?

Reply:
Hello @kobyhallx 

Should we save only the `tar.gz` extension? Or should we save files with `tar.gz` and `zip` extensions?

Reply:
This PR should not change anything to do with tar.gz, since we are just adding nightly tags